subprojects/core/src/main/java/org/gradle/internal/buildevents/BuildExceptionReporter.java

```diff
@@ -213,7 +213,8 @@ private void addBuildScanMessage(BufferingStyledTextOutput resolution) {
 
     private void writeGeneralTips(StyledTextOutput resolution) {
         resolution.println();
-        resolution.text("* Get more help at https://help.gradle.org");
+        resolution.text("* Get more help at ");
+        resolution.withStyle(UserInput).text("https://help.gradle.org");
```

Comment:
:x: The style `UserInput` sounds misleading here. How we have something like `Bold` instead?

Reply:
I would rather not rename this enum as part of this change.

Reply:
I wasn't thinking to rename the existing one but creating a new one instead.